core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2662,7 +2663,24 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 
 			// Add the current folder
 			$strFolderNameEncoded = StringUtil::convertEncoding(StringUtil::specialchars(basename($currentFolder)), Config::get('characterSet'));
-			$return .= Image::getHtml($folderImg, '').' <a href="' . $this->addToUrl('fn='.$currentEncoded) . '" title="'.StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['selectNode']).'"><strong>'.$strFolderNameEncoded.'</strong></a></div> <div class="tl_right">';
+
+            $strFolderLabel = "<strong>$strFolderNameEncoded</strong>";
+
+            // Call the group callback ($strFolderLabel, $objFolder, $protected, $this)
+            if (\is_array($GLOBALS['TL_DCA']['tl_files']['list']['label']['group_callback']))
```

Comment:
`tl_files` should not be hardcoded here.

```suggestion
            if (\is_array($GLOBALS['TL_DCA'][$this->strTable]['list']['label']['group_callback']))
```